randovania/layout/preset_migration.py

```diff
@@ -593,6 +593,12 @@ def _migrate_v26(preset: dict) -> dict:
     return preset
 
 
+def _migrate_v27(preset: dict) -> dict:
+    if preset["game"] == "prime1" and "phazon_suit" not in preset["configuration"]["hints"].keys():
```

Comment:
The second part of the if isn't needed

Reply:
I wanted the test rdvgames to use it without having to remake the rdvgame entirely